distribution/test/itests/test-itests-ddf/pom.xml

```diff
@@ -294,6 +294,7 @@
             </activation>
             <properties>
                 <solr.script.file>solr.cmd</solr.script.file>
+                <solr.force></solr.force>
```

Comment:
We may not need this line if there isn't a command needed on windows. I thought it would just not do the substitution if this was left out.